js/src/views/ParityBar/accountStore.js

```diff
@@ -47,13 +57,22 @@ export default class AccountStore {
         .map((account) => account.address)
     );
 
+    // Have optimistic UI: https://www.smashingmagazine.com/2016/11/true-lies-of-optimistic-user-interfaces/?utm_source=codropscollective
+    this.setDefaultAccount(address);
+
     return this._api.parity
       .setNewDappsWhitelist(accounts)
       .catch((error) => {
         console.warn('makeDefaultAccount', error);
       });
   }
 
+  loadDefaultAccount () {
```

Comment:
It would probably make sense to add this to the spec.js as well (since the store is more-or-less tested already)